src/pages/Acknowledgements/pageContent.tsx

```diff
@@ -165,13 +170,7 @@ export const pageContent = {
         </Text>
         <Text size="medium" className="mt20">
           <FormattedMessage
-            defaultMessage="If you're unfamiliar with keys and mnemomics, please don't proceed. Your
-              keys will be your responsibility and no one can help you if you lose them."
-          />
-        </Text>
-        <Text size="medium" className="mt20">
-          <FormattedMessage
-            defaultMessage="*Warning: Do not store keys on multiple/backup validators at once. {readMore}"
+            defaultMessage="*Warning: Do not store keys on multiple/backup validators at once {readMore}"
```

Comment:
I wonder whether this warning would be better on the generate keys step. There's already a lot of information on this page.